### DISCUSSION TITLE
"Required Checks" in branch policies.

### DISCUSSION THREAD

**Discussion by AskeHolst** [microsoft/AL-Go#465] [Ideas]
I would love to see some docs on recommended setup of "Required Checks" in branch policies.

We had a requirement on CI/CD Workflow, and that worked well. With 3.0 that disappeared. So we changed to require "Build . - Default" or whatever different builds there can be in  multi-project repo.   
But when using auto-merge, the Post Process will fail due to the PR being merged just before or during the Post Process runs.
So we added Post Process as required. And seems to work.

To sum up, it is not trivial to find out how to set it up and what to change for all repositories when a new version of Al-Go rolls out. Please make sure there are both updated docs and mentions in release notes about braking changes like this.

**Comment by freddydk** [comment]
The Pull Request Handler will have a job called Pull Request Status Check, which is designed to be used for precisely this purpose.
This is currently in preview - will ship to main one of the coming days.

**Reply by freddydk** [reply]
And sorry for any inconvenience - we will try to get everything included in the releasenotes

**Comment by freddydk** [comment]
This was implemented a while back.
